traffic_portal/app/src/common/modules/form/server/form.server.tpl.html

```diff
@@ -112,11 +112,11 @@
 
             <label for="profile" ng-class="{'has-error': hasError(serverForm.profile)}">Profile *</label>
             <div ng-class="{'has-error': hasError(serverForm.profile), 'has-feedback': hasError(serverForm.profile)}">
-                <select id="profile" name="profile" class="form-control" ng-model="server.profileId" ng-options="profile.id as profile.name for profile in profiles" required>
+                <select id="profile" name="profile" class="form-control" ng-model="server.profileNames" multiple ng-options="profile.name as profile.name for profile in profiles" required>
```

Comment:
I'm _pretty_ sure that you can bind this to `server.profileNames[0]` if that property is properly initialized to an array in `traffic_portal/app/src/modules/private/servers/new/index.js`, but I still have to test that. I'll report back here once I have and either confirm or resolve the conversation.

Reply:
Yup, I just checked, and with an initialized property you can bind to `server.profileNames[0]` without the `multiple` attribute.

Reply:
If you do that server.profileNames will be:
```
[
  {
    "0": "selected value"
  }
]
```
instead of
```
[
  "selected value"
]
```

Reply:
Nah. It'll do that if you don't get rid of the `multiple` attribute. I just tested it, it works fine, sends `["selected value"]` as expected. Check out https://github.com/rimashah25/trafficcontrol/pull/5